scripts/build-util.sh

```diff
@@ -15,7 +15,7 @@ case ${ncpu} in
 esac
 
 # Allow user to override MAKE
-[ -n "${MAKE}" ] || MAKE=$(command -v gnumake || command -v gmake || command -v make)
+[ -n "${MAKE+x}" ] || MAKE=$(command -v gnumake || command -v gmake || command -v make)
```

Comment:
What's this change trying to do? If MAKE is set to an empty string, the previous command will set it to a default, whereas this will leave it as an empty string. That seems worse

Reply:
${MAKE} crashes bash when it is not defined and `set -u` is used

Reply:
Got it, can we do
```suggestion
[ -n "${MAKE:+x}" ] || MAKE=$(command -v gnumake || command -v gmake || command -v make)
```
instead to avoid a weird failure mode when MAKE is set but empty

Reply:
> > 1. yes. i get a 6.7x speedup on a bwrc using 20 cores (25 kHz vs 4 kHz)
> 
> @abejgonzalez how many cores does `ferry` have? Will this speed up CI?

@jwright6323 keep in mind that compilation for me took 40% longer when compiling for 20 threads on my test case.

Reply:
but verilator compilation is only < 20% of the total latency in the chisel->verilated flow anyways, so it not really that big of a compile-time hit

Reply:
confirmed you are correct. i've included this